src/app/components/reportesDiarios/encabezadoReportesDiarios.component.ts

```diff
@@ -0,0 +1,248 @@
+    public opcionesAnio: { id: number, nombre: string }[] = [];
+    public anio;
+    public mes;
+
+    // Variables "PlanillaC1"
+    public showPlanillaC1 = false;
+    public fecha: any;
+
+    // Eventos
+    @Output() selected: EventEmitter<any> = new EventEmitter<any>();
+
+    constructor(
+        private plex: Plex,
+        private router: Router,
+        private server: Server,
+        private agendaService: AgendaService,
+        private auth: Auth,
+        private servicioOrganizacion: OrganizacionService,
+        private servicioPrestacion: TipoPrestacionService
+    ) {
+
+    }
+
+    public ngOnInit() {
+        this.plex.updateTitle([{
+            route: '/',
+            name: 'ANDES'
+        }, {
+            name: 'Reportes diarios'
+        }]);
+
+        this.organizacion = null;
+        this.tipoReportes = null;
+        this.mes = null;
+        this.anio = null;
+        this.opcionesReportes = this.getObjTiposReportes();
+        this.loadPrestaciones();
+        this.loadOrganizacion();
+        this.opcionesMes = getObjMeses();
+        this.opcionesAnio = this.getObjAnios();
+    }
+
+    getObjTiposReportes() {
+        let tiposReportes = [
+            {
+                id: 1,
+                nombre: 'Resumen diario mensual'
+            },
+            {
+                id: 2,
+                nombre: 'Planilla C1'
+            }];
+        return tiposReportes;
+    }
+
+    getObjAnios() {
+        let anios = [
+            {
+                id: 1,
+                nombre: moment().subtract(2, 'years').format('YYYY')
+            },
+            {
+                id: 2,
+                nombre: moment().subtract(1, 'years').format('YYYY')
+            },
+            {
+                id: 3,
+                nombre: moment().format('YYYY')
+            }
+        ];
+        return anios;
+    }
+
+    loadOrganizacion() {
+        let query = {
+            activo: 1
+        };
+
+        this.servicioOrganizacion.get(query).subscribe(data => {
+            this.opcionesOrganizacion = data;
+        });
+    }
+
+    loadPrestaciones() {
+        this.servicioPrestacion.get({
+            turneable: 1
+        }).subscribe((data) => {
+            this.opcionesPrestacion = data;
+        });
+    }
+
+    refreshSelection() {
+        this.showPlanillaC1 = false;
+        this.showResumenDiarioMensual = false;
+        this.showBotonImprimir = false;
+    }
+
+
+    onChangeTipoReportes(event) {
+        this.parametros = {};
+        this.parametros['organizacion'] = this.organizacion.id;
+        this.parametros['organizacionNombre'] = this.organizacion.nombre;
+        if (this.tipoReportes !== null) {
+            this.parametros['tipoReportes'] = this.tipoReportes.nombre;
+        }
+    }
+
+
+    getParams() {
+        // organizacion
+        if (this.organizacion !== null && typeof (this.organizacion) !== 'undefined') {
```

Comment:
con poner el if (this.organizacion) alcanzaría.

Reply:
Resuelto! gracias!